README.md

```diff
@@ -1,10 +1,155 @@
+devices and other VMM libraries.
+
+The rust-vmm project is organized as a shared-effort, shared-ownership
+open-source project that includes (so far) contributors from Alibaba, AWS,
+Cloud Base, Crowdstrike, Intel, Google, Red Hat as well as individual
+contributors.
+
+Each virtualization component lies in a separated GitHub repository under the
+rust-vmm organization. The code is organized such that one repository
+corresponds to a
+[Rust crate](https://doc.rust-lang.org/book/ch07-01-packages-and-crates-for-making-libraries-and-executables.html).
+
+## Why rust-vmm
+
+- **Reduce code duplication**. The initial thought with rust-vmm was to create a
+  place for sharing common virtualization components between two existing VMMs
+  written in Rust:
+  [CrosVM](https://chromium.googlesource.com/chromiumos/platform/crosvm/) and
+  [Firecracker](https://github.com/firecracker-microvm/firecracker/). These
+  two projects have similar code for calling KVM ioctls, working with the
+  Virtual Machine memory, virtio implementation and others. TODO.
+- **Faster development for new custom VMMs**. rust-vmm provides a base of
+  virtualization components which are meant to be generic enough to be used by
+  other projects besides CrosVM and Firecracker. One example that often comes
+  up is building a container specific VMM. In doing so, the container VMM can
+  reuse most of the rust-vmm components and just build the glue around these
+  components as well as an appropriate API for interacting with the container.
+- **Security & Testability**. Having independent components corresponding to
+  virtualization components makes it easier to run fuzzing on individual
+  packages. The crate we want to focus on in the near future is
+  [vm-virtio](https://github.com/rust-vmm/vm-virtio) which provides a
+  [virtio](https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=virtio)
+  device implementation. We want to keep a high standard in terms of testing
+  as these virtualization packages are going to be used in production by
+  multiple projects. Each component is individually tested with a set of
+  common integration tests that are responsible for running the unit tests,
+  computing the coverage, coding style and other lint checks. In the future
+  we will test the integration of the rust-vmm components by providing a
+  reference VMM implementation.
+- **Clean interface**. With shared crates you also have to think about an
+  interface flexible enough to be used by multiple VMMs. We have multiple
+  rounds of design reviews up to the point we are confident the interface is
+  clean and re-usable by other projects.
+
+## rust-vmm components
+
+Keep in mind that this status might not always be up to date. For the latest
+status please check individual repositories.
+
+**Components with PRs in fly**
+
+These are the empty repositories that have PRs waiting to be merged.
+TODO
+
+**Components in Development:**
```

Comment:
What about "Existing crates" or "Existing components", with a subsection for the ones published to crates.io?
I'm saying this because even crates that are published to crates.io can still be improved and then a new release comes to update crates.io.

Reply:
Sure, I'll re-phrase.